frog/imports/ui/Editor/ActivityType.jsx

```diff
@@ -0,0 +1,212 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import Quiz from './Quiz.jsx';
+
+
+/*
+Class used for the form to be dynamic and change when the user chooses between the different types
+*/
+export default class ActivityType extends Component {
+
+	constructor(props) {
+		super(props);
+
+    this.state = {
+      lectureURL:"",
+      videoURL:"",
+      nbSelected:0,
+			nbQuiz:0, //need this to have unique keys
+      listQuiz:[],
```

Comment:
From testing the app, I have a problem with the `object` field that you create, because I want that the object field always contains an Object (thing between `{}`) and not a list (thing between `[]`). This is important to be coherent with the type of the objects. The database and the code will be cleaner.
```javascript
{
  "_id":"quiz",
  "type":"Quiz",
  "name":"Quiz",
  "plane":1,
  "object":[
    {"question":"1+1","choices":["1","2","3"],"answer":"2"},
    {"question":"1+2","choices":["1","2","3"],"answer":"3"}
  ]
}
```
I suggest the simple change:
```javascript
{
  "_id": "quiz",
  "type": "Quiz",
  "name": "Quiz",
  "plane": 1,
  "object": {
    "questions": [
      {"question":"1+1","choices":["1","2","3"],"answer":"2"},
      {"question":"1+2","choices":["1","2","3"],"answer":"3"}
    ]
  }
}
```